src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileCreatorTest.java

```diff
@@ -141,7 +144,7 @@ public void should_throw_IdamServiceException_when_user_already_exist() {
     @Test
     public void should_throw_IdamServiceException_when_idam_registration_fail() {
 
-        idamRegistrationInfo = new IdamRegistrationInfo(HttpStatus.BAD_REQUEST);
+        idamRegistrationInfo = new IdamRegistrationInfo(status(BAD_REQUEST).build());
         when(userProfileRepository.findByEmail(any(String.class))).thenReturn(Optional.ofNullable(null));
         when(idamService.registerUser(any(IdamRegisterUserRequest.class))).thenReturn(idamRegistrationInfo);
```

Comment:
prefix all the test classes with test in this class and add verify for all the when conditions

Reply:
verify added wherever required.
Its not mandatory that test case name should start with Test or is it Hmcts mandate ?
Following is the naming convention link.Please go through.
https://dzone.com/articles/7-popular-unit-test-naming

Reply:
Thanks shreedhar for  letting me know the naming conventions and others things. before that please go through email QA assurance: John review comments. I did code reviewed based on that email in my mind.

Reply:
> verify added wherever required.
ok